### PR TITLE
V16 QA Implement new release tag for E2E tests

### PR DESCRIPTION
**Pull request by nhudinh0309**
### Prerequisites

- [ ] I have added steps to test this contribution in the description below

If there's an existing issue for this PR then this fixes

### CODE REVIEW

File: build/azure-pipelines.yml [hunk 1]
```diff
@@ -862,6 +862,176 @@ stages:
               searchFolder: "tests/Umbraco.Tests.AcceptanceTest/results"
               testRunTitle: "$(Agent.JobName)"
 
+      - job:
+        displayName: E2E Regression Tests (SQL Server)
```
Comment (andr317c) [inline build/azure-pipelines.yml]:
```suggestion
        displayName: E2E Release Tests (SQL Server)
```

File: build/azure-pipelines.yml [hunk 2]
```diff
@@ -862,6 +862,176 @@ stages:
...
+          matrix:
+            WindowsPart1Of3:
+              vmImage: "windows-latest"
+              testCommand: "npm run releaseTest -- --shard=1/3"
+            WindowsPart2Of3:
+              vmImage: "windows-latest"
+              testCommand: "npm run releaseTest -- --shard=2/3"
+            WindowsPart3Of3:
+              vmImage: "windows-latest"
+              testCommand: "npm run releaseTest -- --shard=3/3"
```
Comment (andr317c) [inline build/azure-pipelines.yml:882]: We don't have that many release tagged tests, so 2 runners might be fine, and what about linux?

File: tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/Content/ContentWithDropdown.spec.ts [hunk 3]
```diff
@@ -83,7 +83,7 @@ for (const dataTypeName of dataTypeNames) {
   });
 }
```
Comment (andr317c) [inline tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/Content/ContentWithDropdown.spec.ts:85]: Might not be needed, as we also have a test for 'can not publish a mandatory checkbox list with an empty value' with the release tag

File: tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/Content/ContentWithRadiobox.spec.ts [hunk 4]
```diff
@@ -84,7 +84,7 @@ test('can create content with the custom radiobox data type', async ({umbracoApi
   expect(contentData.values[0].value).toEqual(optionValues[0]);
 });
 
-test('can not publish mandatory radiobox with an empty value', async ({umbracoApi, umbracoUi}) => {
+test('can not publish mandatory radiobox with an empty value', {tag: '@release'}, async ({umbracoApi, umbracoUi}) => {
```
Comment (andr317c) [inline tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/Content/ContentWithRadiobox.spec.ts:87]: Might not be needed, as we also have a test for 'can not publish a mandatory checkbox list with an empty value' with the release tag

File: tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/Content/ContentWithTextstring.spec.ts [hunk 5]
```diff
@@ -80,7 +80,7 @@ test('can input text into the textstring', async ({umbracoApi, umbracoUi}) => {
   expect(contentData.values[0].value).toEqual(text);
 });
 
-test('cannot input the text that exceeds the allowed amount of characters', async ({umbracoApi, umbracoUi}) => {
+test('cannot input the text that exceeds the allowed amount of characters', {tag: '@release'}, async ({umbracoApi, umbracoUi}) => {
```
Comment (andr317c) [inline tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/Content/ContentWithTextstring.spec.ts]: Might not be needed as we have another releases test in ContentWithTextArea

**Review by andr317c — CHANGES_REQUESTED** [review]
Had a few comments

File: tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/Content/ContentWithTextstring.spec.ts (hunk 5 above)
Comment (nhudinh0309) [reply tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/Content/ContentWithTextstring.spec.ts]: Ok, I removed it

File: tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/Content/ContentWithDropdown.spec.ts (hunk 3 above)
Comment (nhudinh0309) [reply tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/Content/ContentWithDropdown.spec.ts:85]: I think it's still needed, since the validation logic for these property editors is handled differently. We already had two separate issues reported for these two types, so it makes sense to keep both tests.

File: tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/Content/ContentWithRadiobox.spec.ts (hunk 4 above)
Comment (nhudinh0309) [reply tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/Content/ContentWithRadiobox.spec.ts:87]: I think it's still needed, since the validation logic for these property editors is handled differently. We already had two separate issues reported for these two types, so it makes sense to keep both tests.

File: build/azure-pipelines.yml (hunk 2 above)
Comment (nhudinh0309) [reply build/azure-pipelines.yml:882]: I would still prefer to keep 3 runners, since even though the release tests are about 20 fewer than smoke tests, they still take roughly the same amount of time to run.
As for Linux, I'll add it later once the Linux issue has been fixed.
